apis/clusters/v1beta1/cassandra_types.go

```diff
@@ -181,9 +183,10 @@ func (cs *CassandraSpec) HasRestore() bool {
 	return false
 }
 
-func (cs *CassandraSpec) NewDCsUpdate() models.CassandraClusterAPIUpdate {
+func (cs *CassandraSpec) ToClusterUpdate() models.CassandraClusterAPIUpdate {
```

Comment:
```suggestion
func (cs *CassandraSpec) DCsUpdateToInstAPI() models.CassandraClusterAPIUpdate {
```
The Instaclustr APIv2 has multiple endpoints for updating various fields in a cluster specification. One of them is `ClusterSettings`. Therefore, I suggest specifically naming the update fields.

Reply:
Fixed